package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "patternomaly",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
@miller45 Can you also leave the version number as 1.3.0. Changing this on the branch could conflict with another feature being merged and tagged.

Reply:
reverted it in new commit